WikiWatershed/model-my-watershed
# Add firewall rules for CINERGI service

## Overview

Ensure that application servers can send egress traffic (through the NAT) over 8080, which is the port the CINERGI service is bound to.

Fixes #1968 

## Testing Instructions

While these changes are deployed to staging, navigate to https://staging.portal.bigcz.org/search and ensure that the search functionality works.
